man/assign.Rd

```diff
@@ -92,12 +92,12 @@ Since \code{[.data.table} incurs overhead to check the existence and type of arg
 \value{
 \code{DT} is modified by reference and returned invisibly. If you require a copy, take a \code{\link{copy}} first (using \code{DT2 = copy(DT)}).
 }
-\seealso{ \code{\link{data.table}}, \code{\link{copy}}, \code{\link{setalloccol}}, \code{\link{truelength}}, \code{\link{set}}, \code{\link{.Last.updated}}
+\seealso{ \code{\link{data.table}}, \code{\link{copy}}, \code{\link{setalloccol}}, \code{\link{truelength}}, \code{\link{set}}, \code{\link{.Last.updated}},\code{\link{internal.selfref}}
 }
 \examples{
 DT = data.table(a = LETTERS[c(3L,1:3)], b = 4:7)
 DT[, c := 8]                # add a numeric column, 8 for all rows
-DT[, d := 9L]               # add an integer column, 9L for all rows
+DT[, d := 9L]               # add an integer column, 9L for all rows\code{\link{.Last.updated}}
```

Comment:
This syntax doesn't work inside `\examples`. Unfortunately, the Rd format [has a lot of little details](https://developer.r-project.org/parseRd.pdf). Try rendering the help pages using `R CMD Rdconv` or installing the package and browsing its help pages before pushing commits.

Reply:
thanx for the guidance i will introduce all the changes and submit it till toady or tomorrow evening